plugins/janus_videoroom.c

```diff
@@ -952,6 +956,10 @@ int janus_videoroom_init(janus_callbacks *callback, const char *config_path) {
 			if(rec_dir && rec_dir->value) {
 				videoroom->rec_dir = g_strdup(rec_dir->value);
 			}
+			/* Default not notifying the joining event */
```

Comment:
"By default we don't notify about participants simply joining a room, but only about those that start publishing something"